transform/views/main.py

```diff
@@ -128,6 +132,11 @@ def render_pdf():
 
     except IOError as e:
         return client_error("PDF:Could not render pdf buffer: {0}".format(repr(e)))
+    except Exception as e:
+        survey_id = survey_response.get("survey_id", "N/A")
+        tx_id = survey_response.get("tx_id", "N/A")
+        logger.exception("PDF:Generation failed for survey with id {0} and tx_id {1)", survey_id, tx_id)
```

Comment:
survey_id and tx_id as KV pairs please - error message can just be "PDF generation failed" or something :+1: